.travis.yml

```diff
@@ -44,14 +44,23 @@ script:
 
   elif [ "$TEST_TYPE" = build_website ]
   then
-
+    DOCS_VERSION=${TRAVIS_BRANCH%-stable}
     cd website
     $(which npm) install
     ./setup.sh
-    if [ "$TRAVIS_PULL_REQUEST" = false ] && [ "$TRAVIS_BRANCH" = master ]; then
-      # Automatically publish the website
-      echo "machine github.com login reactjs-bot password $GITHUB_TOKEN" >~/.netrc
-      ./publish.sh
+    if [ "$TRAVIS_PULL_REQUEST" = false ]
+    then
+      if [ "$TRAVIS_BRANCH" = master ]
+      then
+        # Automatically publish the website to /next path
+        echo "machine github.com login reactjs-bot password $GITHUB_TOKEN" >~/.netrc
+        ./publish-gh-pages.sh next
```

Comment:
>We need to figure out a way for existing docs pages to still work. Maybe have them redirect in js to the specific version number?
>http://facebook.github.io/react-native/docs/animations.html#content

Do you mean to keep so that we don't break the bookmarks to `http://facebook.github.io/react-native/docs/animations.html#content` so that it got redirected to `http://facebook.github.io/react-native/stable/docs/animations.html#content`?

Yeah, that is definitely on my list